sdp/processors/datasets/ytdlp/create_initial_manifest.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+from pathlib import Path
+import json
+from sdp.logging import logger
+from sdp.processors.base_processor import BaseParallelProcessor, DataEntry
+
+
+class CreateInitialManifestytdlp(BaseParallelProcessor):
+    """
+    Processor for creating an initial dataset manifest by saving youtube links.
+    Make sure to install yt-dlp tool before funning this code. 
+
+    Tool link: https://github.com/yt-dlp/yt-dlp
```

Comment:
Since you use a 3rd party tool here, could you add a specific check in the script whether the tool is installed or not, and log message for user to install it?